test/__helpers__/mockDependencies.ts

```diff
@@ -12,7 +12,7 @@ function mockDependencies(overrides?: Partial<Dependencies>, options?: {
 		release: '11.5.0',
 		fsAccess: () => Promise.resolve(),
 		pathNormalize: normalize,
-		pathSep: '/',
+		pathSep: sep,
```

Comment:
This means tests work differently depending on the platform we are running on.
So, I prefer to stick to the `'/'` value and switch to the `'\'` value if we test things for Windows.

Reply:
The paths were already being normalized in `getFirstExistingParentPath`, so the tests were already running differently depending on the platform. Normalizing on windows changes `'/'` to `'\'` with `dependencies.pathNormalize`.

This means tests can't be run on Windows runners, which was one of the goals of this PR, since the bug was crashing both [HyperPlay](https://github.com/HyperPlay-Gaming/hyperplay-desktop-client) and [Heroic](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher) when running on Windows.

Reply:
reverted :ok_hand: